client/src/components/board.js

```diff
@@ -52,11 +52,29 @@ class Board extends React.Component {
   }
 
   render() {
+    const copyToClipboard = function copyToClipboard() {
+      document.getElementById('urlInput').value = window.location.href;
+      document.querySelector('#urlInput').select();
+      document.execCommand('copy');
+    };
+    const exportCanvas = function exportCanvas() {
+      const exportedImage = document.getElementById('whiteboard').toDataURL();
+      window.open(exportedImage);
+    };
+    $(document).ready(() => {
+      $('.dropdown-button').dropdown();
+    });
```

Comment:
Let's move this away from jQuery. Anything it can do we can also do in react. Try using some simple state to track this. You can still utilize material's CSS